src/style.css

```diff
@@ -1,3 +1,89 @@
-body {
+html {
+  box-sizing: border-box;
+}
+
+*,
+*::after,
+*::before {
+  box-sizing: inherit;
+}
+
+* {
+  padding: 0;
   margin: 0;
+  border: 0;
+}
+
+a {
+  color: inherit;
+  text-decoration: none;
+}
+
+ul li {
+  list-style: none;
+}
+
+img {
+  max-width: 100%;
+  display: block;
+}
+
+:root {
+  --blue-color: #00acdc;
+  --black-color: #000;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  color: var(--black-color);
+  font-size: 12px;
+  line-height: normal;
+}
+
+header {
+  max-width: 1200px;
+  margin: 0 auto;
+  padding: 0 50px;
+}
```

Comment:
The same
- And "Don't fix container size (if there is no such a requirement)"
![image](https://github.com/mate-academy/layout_moyo-header/assets/63243136/4af7ca15-77ce-482e-bf44-6586827007ef)